generators/entity/templates/server/src/main/java/package/domain/_Entity.java

```diff
@@ -58,8 +58,9 @@
 <% } -%>
 <% if (databaseType == 'sql') { -%>
 @Entity
-@Table(name = "<%= entityTableName %>")<% if (hibernateCache != 'no') { %>
-@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)<% } %><% } %><% if (databaseType == 'mongodb') { %>
+@Table(name = "<%= entityTableName %>")
+<% if (hibernateCache != 'no') { if (hibernateCache == 'infinispan') { %>@Cache(usage = CacheConcurrencyStrategy.READ_WRITE) <% } else { %>
```

Comment:
can we also use `===` and `!==` on conditions. I know existing code has `==` but its old code and we need to change it all so atleast when we add new stuff we need to use `===`